pkg/bindings/stan/encoder.go

```diff
@@ -0,0 +1,56 @@
+package stan
```

Comment:
Note that we have changed a lot of the world, this will end up in `pkg/protocol/stan` as a writer